packages/components/src/snackbar/types.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import type {
+	KeyboardEvent,
+	MouseEvent,
+	MutableRefObject,
+	ReactNode,
+} from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import type { WPElement } from '@wordpress/element';
+
+export type ButtonEvent =
+	| KeyboardEvent< HTMLButtonElement >
+	| MouseEvent< HTMLButtonElement >;
+
+export type Action = {
+	label: string;
+	url: string;
+	onClick?: ( event: ButtonEvent ) => void;
+};
+
+export type Notice = {
+	id: string;
+	spokenMessage: ReactNode;
+	actions: Action[];
+	icon?: WPElement | null;
+	onDismiss?: Function;
+	status: string;
+	content: string;
+	__unstableHTML?: string;
+	isDismissible: boolean;
+	type: string;
```

Comment:
```suggestion
	content: string;
	isDismissible: boolean;
```

Let's remove those `Notice` props, since they're currently not used in this component. We will add them back and type them properly when working on the `Notice` component

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/45472/commits/61f5caa469c97cc78cdccf4190be7b1472312c33